running/ambassador-edge/routing/dark-matter-web/prod.yml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-prod-mapping
+  namespace: ambassador-edge
+spec:
+  host: darkmatter.prod.apps.mvpstudio.org
+  prefix: /
+  service: darkmatter-web.darkmatter-prod
```

Comment:
you don't have a service by this name.

Reply:
I created a service (hopefully) by applying my service definition